### ISSUE TITLE
Acting category for show roles

### ISSUE THREAD

**Issue by wjdp** [newtheatre/history-project#658]
Currently lacking a 'Actor'/'Acting' item for show roles. Nontrivial to implement, should be bundled with a refactor of show code.

**Comment by nhyty** [comment]
What do you meant by this Will???

**Comment by wjdp** [comment]
Internal show role list, used in this dropdown:

![image](https://cloud.githubusercontent.com/assets/1690934/23615640/fac000a8-027e-11e7-8c75-36a15d25a783.png)

**Comment by nhyty** [comment]
Could that not be done by saying if in cast???

**Comment by wjdp** [comment]
Yes, it just needs doing. I remember it being a fiddle though, hence it being left for later.
